python/py-scikit-image/Portfile

```diff
@@ -44,6 +44,16 @@ if {${name} ne ${subport}} {
         depends_run-append  port:py${python.version}-dask \
                             port:py${python.version}-toolz \
                             port:py${python.version}-cloudpickle
+    } elseif {${python.version} == 36} {
+        version             0.16.2
```

Comment:
Why downgrade py36-scikit-image to 0.16.x? Is it not possible to add a py36-tifffile port?

Reply:
See comments on 71b77a5d3b; @Schamschula wasn't interested in adding `py36-tifffile` right away, and also thought `py36-scikit-image` was already been pinned to an older version and so didn't need `py36-tiffile`. (`py36-scikit-image` wasn't pinned, hence this PR.) If a `py36-tifffile` port were to be added, it may need to be pinned to an older version, as tifffile has deprecated support for Python 3.6.